### DISCUSSION TITLE
resend/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#8] [Comments]
# resend/

Resend is a new email API for developers. Unfortunately, they’re just an Amazon SES wrapper.

https://matteosonoio.it/resend/

**Comment by konrad-wila** [comment]
There's lots of posts on Reddit about people being denied SES prod access. It's not very transparent as to what the requirements are to get approval. I created a new AWS account and kept getting denied and after messaging my TAM on my main account got access to SES on my other account. It's likely that you need a paid bill plus other reputation factors before you get approved and it's not only what you tell support. 

Resend being a wrapper on top of SES is not a bad thing. SES has some of the better delivery rates, even with amzn's pooled ip's. Vercel is essentialy the same thing being a wrapper on top of other AWS services. All of these AWS wrappers operate on the basis of cheap and good developer experience when your userbase is small to then rake in money once your thing starts to take off.

What is interesting is how resend is able to give access to SES without asking questions as to how people people will be using their service. Seems like resend is giving out access to SES like candy and I wonder if they are pooling users with their own ip's or adding users to amzn's pooled ip's.
